src/app/features/fss-search/fss-search-results/fss-search-results.component.ts

```diff
@@ -28,7 +28,8 @@ export class FssSearchResultsComponent implements OnChanges {
           batchFileDetails: this.getBatchFileDetails(batches[i]),
           BatchID: { key: 'Batch ID', value: batches[i]['batchId'] },
           BatchPublishedDate: { key: 'Batch published date', value: batches[i]['batchPublishedDate'] },
-          ExpiryDate: { key: 'Batch expiry date', value: batches[i]['expiryDate'] }
+          ExpiryDate: { key: 'Batch expiry date', value: batches[i]['expiryDate'] },
+          allFilesZipSize:batches[i]["allFilesZipSize"]
```

Comment:
Please use single quote delimiter
```suggestion
          allFilesZipSize:batches[i]['allFilesZipSize']
```